composer.json

```
@@ -23,7 +23,7 @@
     ],
     "require": {
         "php": ">=5.6.0",
-        "psr/log": "^1.0"
+        "psr/log": "^1|^2"
```

```suggestion
        "psr/log": "^1.0 || ^2.0"
```
https://getcomposer.org/doc/articles/versions.md#version-range

---

the two expressions are equivalent, however i adjusted them to match your suggestion

---

@r4ndsen Thank you!